app/helpers/application_helper.rb

```diff
@@ -78,6 +78,11 @@ def admin_date(date)
     return "#{exact_date} (#{ago_text})"
   end
 
+  def read_asset_file(asset_name)
+    assets = Rails.application.assets_manifest.find_sources(asset_name)
+    assets.first.force_encoding('utf-8') if assets.first
```

Comment:
I think it should raise an exception if it can't be found, otherwise this would silently fail.